src/tapis-ui/components/jobs/JobLauncher/FieldArray.tsx

```diff
@@ -44,20 +46,42 @@ export function FieldArray<T>({
 
   return (
     <div className={styles.array}>
-      <Collapse title={title} note={`${fields.length} items`}>
-        {fields.map((item, index) => (
-          <div className={styles.item}>
-            {render({
-              item,
-              index,
-              remove: !(index in required) ? () => remove(index) : undefined,
-            })}
-          </div>
-        ))}
-        <Button onClick={() => append(appendData)} size="sm">
-          + {addButtonText ?? ''}
-        </Button>
-      </Collapse>
+      {required.length > 0 && isCollapsable ? (
```

Comment:
I think the isCollapsible property should be deep passed into the Collapse UI component. It can have a default value of false in the Collapse component. While it sounds weird to have a collapse component that doesn't collapse it achieves three things:

- One component controls styles rather than FieldArray replicating the styles from Collapse
- The note is preserved (which I think is still desirable in the header for FieldArray to provide a hint as to the number of items)
- There doesn't have to be two field.map blocks in FieldArray